submit-test-status/action.yml

```diff
@@ -33,58 +37,112 @@ runs:
       echo "-----"
       echo "Job name override: ${{ inputs.job_name_override }} (default job name: $GITHUB_JOB)"
       echo "BQ table name (for testing): ${{ inputs.big_query_table_name }}"
+      echo "Test results file path: '${{ inputs.test_results_file_path }}'"
+      echo "Test event record provided: ${{ inputs.test_event_record != '' }}"
+
+      # Show what the load input step will do
+      if [ -n "${{ inputs.test_results_file_path }}" ]; then
+        echo "Will use file path input method: ${{ inputs.test_results_file_path }}"
+      elif [ -n "${{ inputs.test_event_record }}" ]; then
+        echo "Will use inline data input method"
+      else
+        echo "INFO: No input detected - will skip CI Analytics submission (no test issues to report)"
+      fi
+      echo "-----"
+
+  - name: ensure that jq is available
+    uses: dcarbone/install-jq-action@v3.2.0
+
+  - name: load input
+    shell: bash
+    env:
+      TEMP_DIR: ${{ runner.temp }}/sts
+    run: |
+      mkdir -p $TEMP_DIR
+
+      echo "Checking inputs..."
+      echo "test_results_file_path: '${{ inputs.test_results_file_path }}'"
+      echo "test_event_record length: ${#test_event_record}"
+
+      if [ -n "${{ inputs.test_results_file_path }}" ]; then
+        # Method 1: Use provided file path
+        INPUT_FILE="${{ inputs.test_results_file_path }}"
+        echo "Using file path input method: $INPUT_FILE"
+
+        # Verify the file exists
+        if [ ! -f "$INPUT_FILE" ]; then
+          echo "ERROR: File not found: $INPUT_FILE"
+          exit 1
+        fi
+
+      elif [ -n "${{ inputs.test_event_record }}" ]; then
+        # Method 2: Create file from inline input
+        INPUT_FILE="$TEMP_DIR/input_test_event_record.jsonl"
+        echo "Using inline data input method, creating: $INPUT_FILE"
+
+        cat <<EOF > $INPUT_FILE
+      ${{ inputs.test_event_record }}
+      EOF
```

Comment:
🔧 Indentation here is off, all lines should be indented as the `cat`.

It might not cause any problems like this but if it does, they are probably hard to debug and I'd avoid that.